app/tailwind.config.js

```diff
@@ -51,19 +50,196 @@ module.exports = {
                     DEFAULT: 'hsl(var(--card))',
                     foreground: 'hsl(var(--card-foreground))',
                 },
-                surface: {
-                    DEFAULT: '#1a1a1a',
+
+                /*--- TOKENS –--*/
+                bg: {
+                    active: 'var(--bg-tertiary)',
                     brand: {
-                        DEFAULT: 'var(--surface-surface-brand)',
-                        secondary: 'var(--surface-surface-brand-secondary)',
```

Comment:
Overdefined the variable for bg – removed all of the surfaces and made them bg (also consistent with figma now)